Mathlib/Analysis/NormedSpace/Banach.lean

```diff
@@ -250,9 +256,11 @@ protected theorem quotientMap (surj : Surjective f) : QuotientMap f :=
   (f.isOpenMap surj).to_quotientMap f.continuous surj
 #align continuous_linear_map.quotient_map ContinuousLinearMap.quotientMap
 
-theorem _root_.AffineMap.isOpenMap {P Q : Type*} [MetricSpace P] [NormedAddTorsor E P]
-    [MetricSpace Q] [NormedAddTorsor F Q] (f : P →ᵃ[𝕜] Q) (hf : Continuous f)
-    (surj : Surjective f) : IsOpenMap f :=
+theorem _root_.AffineMap.isOpenMap {𝕜 : Type*} [NontriviallyNormedField 𝕜] {E : Type*}
```

Comment:
Why do you declare `𝕜` and `E` again here? I'm happy with redeclaring `F` since you change the assumption, but for these I don't understand. Am I missing something ?

Reply:
nope, just my mistake!